flax/nn/base.py

```diff
@@ -112,6 +112,15 @@ def path(self):
       path += self.name
     return path
 
+  def is_child_of(self, frame):
```

Comment:
I think the right term is `is_descendent_of`. child typically describes first-order children (the converse of "parent")